crates/wasmtime/src/config.rs

```diff
@@ -1,24 +1,25 @@
-use crate::memory::MemoryCreator;
-use crate::trampoline::MemoryCreatorProxy;
-use anyhow::{bail, Result};
-use serde::{Deserialize, Serialize};
 use std::collections::{HashMap, HashSet};
 use std::fmt;
 #[cfg(feature = "cache")]
 use std::path::Path;
 use std::sync::Arc;
+
+use anyhow::{bail, Result};
+use serde::{Deserialize, Serialize};
 use wasmparser::WasmFeatures;
+
 #[cfg(feature = "cache")]
 use wasmtime_cache::CacheConfig;
+pub use wasmtime_environ::CacheStore;
 use wasmtime_environ::Tunables;
 use wasmtime_jit::{JitDumpAgent, NullProfilerAgent, ProfilingAgent, VTuneAgent};
 use wasmtime_runtime::{InstanceAllocator, OnDemandInstanceAllocator, RuntimeMemoryCreator};
-
-pub use wasmtime_environ::CacheStore;
-
 #[cfg(feature = "pooling-allocator")]
 pub use wasmtime_runtime::{InstanceLimits, PoolingAllocationStrategy};
 
+use crate::memory::MemoryCreator;
+use crate::trampoline::MemoryCreatorProxy;
+
```

Comment:
Here and in some other files there are a lot of changes to the imports here, but I don't think that anything substantively changed? Could the changes you're adding here be backed out to avoid churn in this commit?